### PR TITLE
Very minor updates to the comments in index.d.ts.

### PR DESCRIPTION
**Pull request by adamdonahue**
## CHANGE SUMMARY

* Made a handful of very small changes to the comments to `index.d.ts`, fixing a grammatical error (data is plural), and removing some slightly misleading text.  See inline comments in the PR for details.

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -26,7 +26,7 @@ export interface Action {
  * to reduce a collection of values down to a single value
  *
  * Reducers are not unique to Redux—they are a fundamental concept in
- * functional programming.  Even most non-functional languages, like
+ * functional programming.  Even many non-functional languages, like
```
Comment (adamdonahue) [inline index.d.ts]: I'm not sure `most` is the right text; `many` seems more apt.  There are a lot of programming languages out there.

File: index.d.ts [hunk 2]
```diff
@@ -37,10 +37,6 @@ export interface Action {
  * side-effects. This is what enables exciting features like hot reloading and
  * time travel.
  *
- * Reducers are the most important concept in Redux.
```
Comment (adamdonahue) [inline index.d.ts]: Is this fair to say?  I feel it's misleading to say one concept or another are the "most important" in Redux.  Perhaps saying they are the primary contribution of, or new concept introduced by, Redux would be more apt.

File: index.d.ts [hunk 3]
```diff
@@ -37,10 +37,6 @@ export interface Action {
...
- *
- * *Do not put API calls into reducers.*
```
Comment (adamdonahue) [inline index.d.ts]: This is far too broad a statement.  As long as those APIs are also pure and side-effect free there is no reason not to use them in a reducer.  (For example, for handling complex data arithmetic one might want to use the `moment` API.)

File: index.d.ts [hunk 4]
```diff
@@ -295,7 +291,7 @@ export function applyMiddleware(...middlewares: Middleware[]): GenericStoreEnhan
 
 /**
  * An *action creator* is, quite simply, a function that creates an action. Do
- * not confuse the two terms—again, an action is a payload of information, and
+ * not confuse the two terms—an action is a payload of information, and
```
Comment (adamdonahue) [inline index.d.ts]: Saying "again" implies this was said before, which is wasn't.

**Comment by adamdonahue** [comment]
Did I mention these are _extremely minor_?

File: index.d.ts (hunk 3 above)
Comment (gaearon) [inline index.d.ts]: Can change this to “side effects such as AJAX calls”.

Comment (adamdonahue) [inline index.d.ts]: Made a small change.  What do you think?

**Comment by gaearon** [comment]
Hi, did you mean to close this? I’m open to merging it.

**Comment by adamdonahue** [comment]
I did. I'll reopen with the cleaner changes.

**Pull request closed without merging**